Alt and link text edits - Message Building by Channel - Pt. 2 (#3275)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/email/creating_an_email_campaign.md

When you're ready for a final check, select **Test Send** and send a test message to yourself or a group of content testers to ensure that your email displays properly on a variety of devices and email clients.

> ![Image that shows the Test Send option and an example email preview.][15]

Reviewer: ```suggestion
![Test Send option and example email preview when composing your email.][15]
```
